src/app/main/component/events/components/event-details/event-details.component.scss

```diff
@@ -73,7 +73,6 @@
           font-weight: 600;
           font-size: 16px;
           line-height: 24px;
-          color: var(--primary-green);
```

Comment:
According to [task](https://github.com/ita-social-projects/GreenCity/issues/5129) you should just add `cursor: pointer`